requirements.md

```diff
@@ -66,10 +66,49 @@ These are mostly in response to trends we see in issues made
 - Switch between slower HiFi resampler and a fast LoFi one.
 
 ### Needed by Bevy
-Bevy is a widely used game engine build around the ECS model. Because bevy uses ECS it has specific demands from an audio engine.
 
-*TODO*
+Bevy is a popular game engine built around the ECS paradigm.
+Its ECS is highly flexible, so there are many valid ways to integrate
+audio engines. However, idiomatic integrations
+should strive to satisfy three conditions:
 
+1. ECS-driven design.
+2. Broad platform support.
+3. Integration with Bevy's asset system.
+
+#### ECS-driven design
+
+Audio engines are a good fit for ECS-driven design. Playing sounds
+can be modeled by spawning sound entities. Audio tracks and buses
+can be modeled as groups of related entities. Depending on the
+engine, it may also make sense to model effects as entities.
+
+Many audio engines will facilitate ECS-driven integrations out of the box.
+The only hard requirement is that audio types must implement Bevy's 
+[`Component`](https://docs.rs/bevy/latest/bevy/ecs/component/trait.Component.html)
+trait. This can be done directly, through a newtype or other wrapper,
+or even via managed proxy types.
+
+#### Broad platform support
+
+Bevy features broad platform support, including
+desktop platforms, mobile, WebAssembly, and increasingly
+resource-constrained environments such as microcontrollers.
```

Comment:
I don't think it is necessarily straightforward that users would expect an audio engine to work seamlessly on embedded platforms, similarly to the graphics engine for example. Embedded is very hard to support because of the multitude of setups and solutions, and even with ecosystem-aligning crates like `embassy` it is still going to depend on hardware availability and setup.

I would be in favor of not meeting the needs of embedded projects, at least at first, as I feel this would open up a can of scope-creep worms, and admittedly, I don't know the embedded audio ecosystem, so if anybody knows more about the ecosystem, maybe this can be eventually included.

Reply:
Is this a serious requirement for Bevy? Borrowing from @cart, if we do not meet this does that mean Bevy will eventually adopt its own thing? Or are embedded games simple enough users will be happy to use a specialized embedded solution like https://github.com/10buttons/awedio?

I'll open a separate PR to discuss the use case of embedded playback. Like a low power tiny portable music player.

Reply:
No, embedded platforms are not particularly important. Such support would be nice, but obviously low priority. I can reword this to remove that perception.

Reply:
Yeah, embedded is not particularly important to us. Definitely solidly in the nice-to-have realm.